src/ctr_logging.c

```diff
@@ -126,6 +137,12 @@ void configure_log_drivers(gchar **log_drivers, int64_t log_size_max_, char *cuu
 			/* save the length so we don't have to compute every sd_journal_* call */
 			name_len = strlen(name);
 			container_name = g_strdup_printf("CONTAINER_NAME=%s", name);
+
+			syslog_identifier = g_strdup_printf("SYSLOG_IDENTIFIER=%s", name);
+			syslog_identifier_len = name_len + SYSLOG_IDENTIFIER_EQ_LEN;
+		} else {
+			syslog_identifier = g_strdup_printf("SYSLOG_IDENTIFIER=%s", short_cuuid);
+			syslog_identifier_len = TRUNC_ID_LEN + SYSLOG_IDENTIFIER_EQ_LEN;
```

Comment:
same here

Reply:
the one thing I am concerned about is the null byte here, but in other uses we use `TRUNC_ID_LEN + CID_EQ_LEN` which doesn't seem to count null